### DISCUSSION TITLE
Didn't find class "com.google.android.exoplayer2"

### DISCUSSION THREAD

**Discussion by Zelfapp** [jhomlala/betterplayer#602] [Q&A]
Updated today to `0.0.72`. After running `flutter clean` and launching main.dart I'm getting the following exoplayer2 error.

I don't know that betterplayer caused this, but my question is how do I resolve it? When I go to a page with either video_player or betterplayer my app now fatally crashes. I've tried several solutions out there for build.gradle updates, etc., but nothing is working. I tried reverting the betterplayer version to `0.0.70`, which I was running just fine before the upgrade, but the same error persists.

```
E/AndroidRuntime( 8329): Caused by: java.lang.ClassNotFoundException: Didn't find class "com.google.android.exoplayer2.ExoPlayerFactory"
```

More logs

```
E/AndroidRuntime( 8329): FATAL EXCEPTION: main
E/AndroidRuntime( 8329): Process: com.app.com, PID: 8329
E/AndroidRuntime( 8329): java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/android/exoplayer2/ExoPlayerFactory;
```

**Comment by Zelfapp** [comment]
Disregard, the issue was with another package.

**Reply by jhomlala** [reply]
Please don't use other video players with Better Player. This may lead to issues like this.
